ext/opentelemetry-ext-opentracing-shim/src/opentelemetry/ext/opentracing_shim/__init__.py

```diff
@@ -0,0 +1,289 @@
+            return None
+
+        wrapped_span = SpanWrapper(self._tracer, span.get_context(), span)
+        return ScopeWrapper(self, span=wrapped_span)
+        # TODO: Return a saved instance of SpanWrapper instead of constructing
+        # a new object (and the same for ScopeWrapper?).
+        # https://github.com/open-telemetry/opentelemetry-python/issues/161#issuecomment-534136274
+
+    @property
+    def tracer(self):
+        return self._tracer
+
+
+class TracerWrapper(opentracing.Tracer):
+    def __init__(self, tracer, scope_manager=None):
+        scope_manager = ScopeManagerWrapper(self)
+        super().__init__(scope_manager=scope_manager)
+        self._otel_tracer = tracer
+
+    def unwrap(self):
+        """Returns the wrapped OpenTelemetry `Tracer` object."""
+
+        return self._otel_tracer
+
+    def start_active_span(
+        self,
+        operation_name,
+        child_of=None,
+        references=None,
+        tags=None,
+        start_time=None,
+        ignore_active_span=False,
+        finish_on_close=True,
+    ):
+        span = self.start_span(
+            operation_name=operation_name,
+            child_of=child_of,
+            references=references,
+            tags=tags,
+            start_time=start_time,
+            ignore_active_span=ignore_active_span,
+        )
+        return self._scope_manager.activate(span, finish_on_close)
+
+    def start_span(
+        self,
+        operation_name=None,
+        child_of=None,
+        references=None,
+        tags=None,
+        start_time=None,
+        ignore_active_span=False,
+    ):
+        if child_of is None:
+            parent = None
+        else:
+            if isinstance(child_of, (SpanWrapper, SpanContextWrapper)):
+                # The parent specified in `child_of` is valid and is either a
+                # `SpanWrapper` or a `SpanContextWrapper`. Unwrap the `Span` or
+                # `SpanContext` to extract the OpenTracing object and use this
+                # object as the parent of the created span.
+                parent = child_of.unwrap()
+            else:
+                logger.warning(
+                    "Unknown class %s passed in child_of argument to start_span() method.",
+                    type(child_of),
+                )
+                parent = None
+                # TODO: Refuse to create a span and return `None` instead of
+                # proceeding with a `None` parent? This would cause the created
+                # span to become a child of the active span, if any, or create
+                # a new trace and make the span the root span of that trace.
+
+        # Use active span as parent when no explicit parent is specified.
+        if (
+            self.active_span is not None
+            and not ignore_active_span
+            and not parent
+        ):
+            parent = self.active_span.unwrap()
+
+        span = self._otel_tracer.create_span(operation_name, parent)
```

Comment:
As I had mentioned, `create_span()` is probably going away (I will have a PR by tomorrow). Any reason to **not** use `start_span()`?

Reply:
Please see https://github.com/open-telemetry/opentelemetry-python/pull/211#discussion_r335018567.

To summarize, if we switch to `start_span()` right now, it will introduce the following limitations:

- Can't handle `finish_on_close`. We will only be able to have a hardcoded value there.
- Can't set references and tags on a span object before starting the span. Not sure if this is a problem.
- Can't set an explicit start time for a span.

Reply:
> To summarize, if we switch to start_span() right now, it will introduce the following limitations:

I see no reason for this, but let me work on that then, after this is merged, and when I create a PR for removing `create_span()` ;)

Reply:
OK, so I'm leaving `create_span()` in the bridge's implementation for now.